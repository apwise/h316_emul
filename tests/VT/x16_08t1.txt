ptr ../../tapes/VT/X16-08T1_slst.ptp
g'1
cl
# Load A with 4 (32K of memory)
m'1001,'005777
m'1777,4
# Make end of pass jump to the TYPO routine
m'2040,'003066
# Run continously (don't halt each pass)
m'2060,'140040
#
cl
limit 120000
g'1000
q
