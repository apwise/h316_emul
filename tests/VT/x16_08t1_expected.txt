
MON> ptr ../../tapes/VT/X16-08T1_slst.ptp
MON> g'1

MON> cl
MON> # Load A with 4 (32K of memory)
MON> m'1001,'005777
MON> m'1777,4
MON> # Make end of pass jump to the TYPO routine
MON> m'2040,'003066
MON> # Run continously (don't halt each pass)
MON> m'2060,'140040
MON> #
MON> cl
MON> limit 120000
MON> g'1000

X16-08T1 PASS 000000001      
X16-08T1 PASS 000000002      
0003939443: limit reached

MON> q
